From: Free Ekanayaka <freee@debian.org>
Date: Mon, 6 Feb 2017 12:10:17 +0000
Subject: skip-phantomjs-tests

Skip PhantomJS tests since the phantomjs package in Debian does
not support running headless.
---
 tests/test_phantomjs.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/test_phantomjs.py b/tests/test_phantomjs.py
index 1407acc..09e58ec 100644
--- a/tests/test_phantomjs.py
+++ b/tests/test_phantomjs.py
@@ -1,4 +1,4 @@
-from unittest import skipIf
+from unittest import skip
 
 from testtools import TestCase
 
@@ -28,7 +28,7 @@ class PhantomJSIntegrationTest(TestCase):
 
         self.fixture = PhantomJS(reactor, timeout=10)
 
-    @skipIf(not hasTwist, "twist executable not available")
+    @skip("Debian's phantomjs can't run headless")
     def test_webdriver(self):
         """After setUp is run, the service is fully ready."""
         self.useFixture(self.fixture)
